source/misc/datapacks.md

```diff
@@ -0,0 +1,338 @@
+---
+title: Data Packs Customization
+layout: default
+---
+# Data Pack Customization
+
+- [Concepts](#concepts)
+  - [Terminology](#terminology)
+  - [Example Folder Layout](#example-folder-layout)
+- [Tags](#tags)
+  - [Block Tags](#block-tags)
+  - [Item Tags](#item-tags)
+  - [Crafter Item Tags](#crafter-item-tags)
+- [Crafter Recipes](#crafter-recipes)
+- [Player Recipes](#player-recipes)
+- [Loot Tables](#loot-tables)
+- [Research Customization](#research-customization)
+  - [Branches](#branches)
+  - [Research](#research)
+  - [Research Effects](#research-effects)
+  - [MineColonies Research Effects](#minecolonies-research-effects)
+  - [MineColonies Building Unlocks](#minecolonies-building-unlocks)
+
+MineColonies allows modifications of many features using data packs, including player and worker recipes, loot tables, and mob drops. This allows broad expansion by players or modpack makers to support other mods, design choices, forms of progression, or styles of play. For general information on MineCraft Data Packs, [see the official wiki](<https://minecraft.fandom.com/wiki/Data_Pack>).
+
+Data Packs exist as part of a world, and they must be [installed](https://minecraft.fandom.com/wiki/Tutorials/Installing_a_data_pack) on each world.
+
+## Concepts
+
+Data Packs are one or more files in the JSON format, stored within a folder or a zip file. Despite their names, these are text files, and can be opened with text editors. Note that Windows will, by default, hide the extension of known files, and this [should be changed](https://support.microsoft.com/en-us/windows/common-file-name-extensions-in-windows-da4a4430-8e76-89c5-59f7-1cdbbc75cb01) to avoid accidentally appending .txt to the file name. Avoid using a rich text editor like WordPad or Microsoft Word, which may insert additional formatting into the file. On Windows, simple text editors like Notepad, NotePad++, or WattPad are more useful for making small numbers of these files, and development environments like IntelliJ may be worth installing if creating many JSONs.
```

Comment:
```suggestion
Data packs are one or more files in the JSON format, stored within a folder or a zip file. Despite their names, these are text files, and can be opened with text editors. Note that Windows will, by default, hide the extension of known files, and this [should be changed](https://support.microsoft.com/en-us/windows/common-file-name-extensions-in-windows-da4a4430-8e76-89c5-59f7-1cdbbc75cb01) to avoid accidentally appending .txt to the file name. Avoid using a rich text editor like WordPad or Microsoft Word, which may insert additional formatting into the file. On Windows, simple text editors like Notepad, NotePad++, or WattPad are more useful for making small numbers of these files, and development environments like IntelliJ may be worth installing if creating many JSONs.
```